hypothesis-python/src/hypothesis/internal/validation.py

```diff
@@ -82,7 +82,7 @@ def try_convert(typ, value, name):
         return value
     try:
         return typ(value)
-    except (TypeError, OverflowError, ValueError, ArithmeticError):
+    except (TypeError, ValueError, ArithmeticError):
```

Comment:
I'm particularly pleased that `flake8-bugbear` caught this one, because I wrote the check for redundant entries in an except-clause for the benefit of my students... and it turned out that we hit it too :joy: